src/timer.cr

```diff
@@ -0,0 +1,28 @@
+module SDL
+  module Timer
+    
```

Comment:
Why not just on this line `extend self` instead of writing `self.` at every single method

Reply:
I prefer explicit `self.` definitions over `extend self`, which only has value if the module is meant to be used as an includable module and a namespace for global methods at the same time —it may be valid here.

Reply:
Ditto ^ 😂